datasources/04_ACT_Government.json

```diff
@@ -1,7 +1,15 @@
 {
   "catalog": [
     {
-      "items": [],
+      "items": [
+        {
+          "type": "socrata",
+          "name": "data ACT",
```

Comment:
There are potentially other sources of ACT data which I haven't really investigated, but I think you're right in the short-medium term.